_posts/2023-11-15-Codesgiving-Part-1.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Tip of the Week: Codesgiving - Open-source Contribution Walkthrough"
+author: dave-bunten
+tags:
+  - tip-of-the-week
+  - codesgiving
+---
+
+# Tip of the Week: Codesgiving - Open-source Contribution Walkthrough
+
+{% include tip-of-the-week-intro.html %}
+
+## Introduction
+
+<!-- excerpt start -->
+[Thanksgiving](https://en.wikipedia.org/wiki/Thanksgiving) is a holiday practiced in many countries which focuses on gratitude for good harvests of the preceding year.
+In the United States, we celebrate Thanksgiving on November 23rd each year often by eating meals we create together with others.
```

Comment:
I think it's on the fourth Thursday each year so not always the 23rd?

Reply:
Thanks, oops, going to correct this!